### ISSUE TITLE
.header and .footer are too generic

### ISSUE THREAD

**Issue by vladrusu** [coreui/coreui#849]
Please considering adding prefixes to these classes (can be added to _variables.scss), or rename them to .site-header or .site-footer, or something like that.

I find that ".header" and ".footer" classes are WAY too generic as names.
I have a project that has multiple .header classes spawn all over the code (cards, different containers etc.). Using coreui basically breaks my code.
I think that anything that's a container can have a .header and .footer class that SHOULD NOT be used, as-is, by a UI framework.

**Comment by mrholek** [comment]
@vladrusu, you're right. Several months ago, we decided to abandon that component in v6. The header duplicates the navbar, and the footer can be replaced with utilities. In v6, we also want to introduce something called "Design Blocks" with extended examples of headers and footers.
